src/lib/components/workflow/workflows-summary-configurable-table/workflows-summary-configurable-table.svelte

```diff
@@ -0,0 +1,471 @@
+  let resizableContainerWidth: number = $workflowPinnedColumnsWidth;
+  let resizing: boolean = false;
+
+  $: namespace = $page.params.namespace;
+  $: selectedWorkflowsCount = selectedWorkflows.length;
+  $: showBulkActions = selectedWorkflowsCount > 0;
+  $: terminateEnabled = workflowTerminateEnabled($page.data.settings);
+  $: cancelEnabled = workflowCancelEnabled($page.data.settings);
+
+  $: sortDisabled =
+    $workflowCount?.totalCount >= 1000000 ||
+    !$supportsAdvancedVisibilityWithOrderBy;
+
+  $: indeterminate =
+    selectedWorkflows.length > 0 && selectedWorkflows.length < workflows.length;
+
+  $: namespaceWriteDisabled = $coreUser.namespaceWriteDisabled(
+    $page.params.namespace,
+  );
+
+  $: checked =
+    allSelected ||
+    pageSelected ||
+    (selectedWorkflowsCount === workflows.length &&
+      selectedWorkflowsCount !== 0);
+
+  $: pinnedColumns = $workflowTableColumns.filter((column) => column.pinned);
+
+  $: otherColumns = $workflowTableColumns.filter((column) => !column.pinned);
+
+  const openCustomizationDrawer = () => {
+    customizationDrawerOpen = true;
+  };
+
+  const closeCustomizationDrawer = () => {
+    customizationDrawerOpen = false;
+  };
+
+  const handleSelectAll = (event: MouseEvent | KeyboardEvent) => {
+    if (
+      event instanceof MouseEvent ||
+      (event instanceof KeyboardEvent && event.key === 'Enter')
+    ) {
+      dispatch('selectAll');
+    }
+  };
+
+  const handleCheckboxChange = (event: CustomEvent<{ checked: boolean }>) => {
+    const { checked } = event.detail;
+    dispatch('togglePage', { checked, ...(checked && { workflows }) });
+  };
+
+  const goToWorkflow = (workflow: WorkflowExecution) => {
+    goto(
+      routeForEventHistory({
+        namespace,
+        workflow: workflow.id,
+        run: workflow.runId,
+      }),
+    );
+  };
+
+  const handleMouseDown = (event: MouseEvent) => {
+    if (resizableContainer.clientWidth - event.x < 3) {
+      resizing = true;
+    }
+    return false;
+  };
+
+  const handleMouseUp = () => {
+    if (resizing) resizing = false;
+    $workflowPinnedColumnsWidth = resizableContainerWidth;
+  };
+
+  const handleMouseMove = (event: MouseEvent) => {
+    if (!resizing) return false;
+    const rect = resizableContainer.getBoundingClientRect();
+    resizableContainerWidth = event.x - rect.x;
+    return false;
+  };
+</script>
+
+<svelte:window
+  on:mousemove|stopPropagation={handleMouseMove}
+  on:mouseup|stopPropagation={handleMouseUp}
+/>
+<div class="workflow-summary-tables-wrapper">
+  <div
+    class="workflow-summary-table-wrapper pinned"
+    class:batch-actions-enabled={$supportsBulkActions}
+    class:batch-actions-visible={showBulkActions}
+    class:no-columns-pinned={pinnedColumns.length === 0}
+    bind:this={resizableContainer}
+    style="width:{resizableContainerWidth}px;"
+    on:mousedown|stopPropagation|preventDefault={handleMouseDown}
+  >
+    <table class="workflow-summary-table pinned">
```

Comment:
It gets a little wonky on smaller screen sizes ⬇️ 

![Screenshot 2023-03-27 at 3 52 23 PM](https://user-images.githubusercontent.com/15069288/228078093-5c0a69f4-3270-4965-98c8-aa46a512b747.png)

Reply:
Nice catch, should be fixed: [77446df](https://github.com/temporalio/ui/pull/1229/commits/77446df02b0101b19d63bd49364dfd5627d6e925)